std_msgs/Float64 time
std_msgs/Int8 controller
geometry_msgs/PoseStamped pose
geometry_msgs/Twist velocity
geometry_msgs/Twist acceleration
